src/components/ProgressData.vue

```diff
@@ -94,19 +106,32 @@
             },
         },
         methods: {
-            ...mapActions(['importProgress']),
+            ...mapActions(['importProgress', 'loadProgress']),
+            getProgress() {
+                return {
+                    funds: this.$store.getters.funds,
+                    stockPortfolio: this.$store.getters.stockPortfolio,
+                    stocks: this.$store.getters.stocks,
+                };
+            },
             getImageUrl(filename) {
                 return require(`../assets/stocks/${filename}`);
             },
             handleImportProgress() {
                 this.importProgress();
             },
+            handleLoadProgress() {
+                this.loadProgress();
+            },
+            handleSaveProgress() {
+                const progress = this.getProgress();
+
+                localStorage.setItem('progress', JSON.stringify(progress));
+                alert('Your progress is saved');
```

Comment:
Probably, later I will implement a nice toaster, but for the moment it is ok to keep it like this... 😄